### DISCUSSION TITLE
希望增加：1.聊天式的交互逻辑 2.网页端连接手机服务器

### DISCUSSION THREAD

**Discussion by mufengyouyang** [localsend/localsend#789] [L-简体中文]
1.通常局域网传输，采用qq和微信这样子聊天框的形式，比较直观和便捷。
现在的形式虽然不错，但是操作略微繁琐，需要先选择发送类型，然后编辑内容，再点击一下发送对象，比较麻烦。
如果采用qq或者微信那样子，聊天的形式，不需要考虑发送什么类型，直接把要发送的文字，文件，文件夹复制粘贴上去，直接按一下回车就发送，这样的形式直观且便捷。

2.电脑端的适配其实每一次做起来会增加工作负担，不如直接将手机端设置为服务器，电脑端直接打开浏览器输入手机服务器的ip+端口，直接在浏览器上传输，这样普适性会更广。
已经有的案例有：局域网快传、速享（不过他们有个缺陷，电脑端将图片发送到手机后，手机qq等应用无法找到并且发送给别人）

**Comment by cmlanche** [comment]
https://lightningvine.zishu.life/download.html  用这个吧

**Reply by mufengyouyang** [reply]
做的也不好，手机端的按钮太小了，电脑段免安装版链接手机，显示灰屏。
最大的痛点是发送前要选择发送的文件类型，只有采用聊天框的发送形式是最好的。
不用考虑什么 类型，一股脑发送到手机就行了。

**Reply by jaceDeng** [reply]
类似飞鸽传书 如果能 静默收到后 发个气泡提示就好了

**Reply by ghost** [reply]
> 做的也不好，手机端的按钮太小了，电脑段免安装版链接手机，显示灰屏。
> 最大的痛点是发送前要选择发送的文件类型，只有采用聊天框的发送形式是最好的。
> 不用考虑什么 类型，一股脑发送到手机就行了。

可以使用360家的极连快传，它实现了聊天框式的发送形式。不过会经过他家的服务器，好处是支持了广域网传输，不过广域网的速度挺慢。
有时局域网传输也会发疯，遇到两次局域网文件传输失败，或许是文件太大。

**Comment by sunshine681** [comment]
聊天框的发送形式很直观高效。建议开发者参考一下全平台的Feem。比如我手机相册里面有三张图片要发到iPad，直接选择图片以后，点分享→点Feem→选iPad，就跟微信一样快捷。比LocalSend方便快捷多了。

**Reply by Ardenet** [reply]
localsend也可以做到你说的通过分享发出去吧，我一直这么干的呀🤔🤔
我唯一希望的就是可以让它在手机后台保活，不用每次传东西的时候都要打开

**Comment by Mogez** [comment]
同求，传统聊天界面即可，像 QQ、Messenger、飞秋那样。
这个软件的发送逻辑是先准备素材，再选发送目标，正好和习惯相反。采用古老的弹窗方式作消息提醒，也没有对话上下文，如果开了快速接收要切到历史记录去找。
楼上提到的闪电藤好像还行，加了些收费功能（其实也没什么用）。先试用看看。

**Reply by cmlanche** [reply]
确实很多收费功能大部分用户用不着，很多支持都算是打赏了

**Reply by cmlanche** [reply]
网页端连服务器，有个问题就是，闪电藤是基于ls的，他们都是http协议的服务器，无法双向传输，所以目前网页版也只能单向的，只能网页版给其他端发送文件，其他端无法给网页版发
